fedora/Dockerfile

```diff
@@ -1,13 +1,9 @@
-FROM fedora:20
+FROM fedora
 MAINTAINER amitsaha.in@gmail.com
 
-# Let's start with some basic stuff.
-RUN yum -y clean all
-RUN yum -y update
-RUN yum install -y iptables ca-certificates lxc e2fsprogs
-
-# Install Docker from Fedora repos
-RUN yum -y install docker-io
+# Update and install docker
+RUN dnf -y update && dnf clean all
+RUN dnf -y install docker && dnf clean all
```

Comment:
Fair enough, I can see the point of not caching an update separate from installation.  Done.